lib/clif-backend/src/signal/unix.rs

```diff
@@ -76,7 +76,7 @@ pub fn call_protected<T>(handler_data: &HandlerData, f: impl FnOnce() -> T) -> R
             *jmp_buf = prev_jmp_buf;
 
             if let Some(data) = super::TRAP_EARLY_DATA.with(|cell| cell.replace(None)) {
-                Err(RuntimeError::Error { data })
+                Err(RunErr::Error(data))
```

Comment:
I kind of like more `RuntimeError`? But I'm good with either. Maybe @MarkMcCaskey  or @xmclark can provide more input here.

Reply:
It's not the same enum actually. `RuntimeError` is from the `runtime_core`, but `RunErr` is a local enum that's just used in the interaction between `call_protected` and the invoke function.

Reply:
`CallProtectedError`

Reply:
Ok, will rename.